### DISCUSSION TITLE
Plans for edge weights and edge features in the `FeatureStore`?

### DISCUSSION THREAD

**Discussion by denadai2** [pyg-team/pytorch_geometric#8837] [Q&A] [answered]
Dear PyG,

I am running large scale experiments on PyG. However, I've noticed that the library doesn't seem to support edge features at the moment. Could you share if there are any plans to integrate this feature into PyG in the near future (e.g. if you already know the timeline etc)? Additionally, I would be grateful for any advice or suggestions you could offer on implementing edge weights at scale!

**Comment by rusty1s** [comment]
Edge features are well supported, and there exists a variety of GNN layers that can utilize them, see [here](https://pytorch-geometric.readthedocs.io/en/latest/cheatsheet/gnn_cheatsheet.html).

**Reply by denadai2** [reply]
thank you!

**Comment by denadai2** [comment]
Sry I meant in the distributed featurestore ThxOn 31 Jan 2024, at 19.42, Matthias Fey ***@***.***> wrote:﻿
Edge features are well supported, and there exists a variety of GNN layers that can utilize them, see here.

**Reply by rusty1s** [reply, marked as answer]
Got it. This will still take some time to implement. Bumping its priority.
